src/phoenix/server/api/dataloaders/experiment_annotation_summaries.py

```diff
@@ -37,43 +35,48 @@ def __init__(
     async def _load_fn(self, keys: list[Key]) -> list[Result]:
         experiment_ids = keys
         summaries: defaultdict[ExperimentID, Result] = defaultdict(list)
+        repetition_scores_subquery = (
+            select(
+                models.ExperimentRunAnnotation.experiment_run_id.label("experiment_run_id"),
```

Comment:
If you want weighted average, you should group by dataset_example_id instead.
```suggestion
                models.ExperimentRun.dataset_example_id,
```